client/my-sites/media-library/filter-bar.jsx

```diff
@@ -63,6 +66,8 @@ export class MediaLibraryFilterBar extends Component {
 		const { translate } = this.props;
 
 		switch ( filter ) {
+			case 'this-post':
+				return translate( 'This Post', { comment: 'Filter label for media list' } );
```

Comment:
Actually, I just checked the core behavior and it says `post` regardless of post type (including CPTs), so this is probably fine for the purposes of this PR.